### PR TITLE
changing the warning to include the project/workspace folder 

### LINKED ISSUE #5425: Pytest: Unexpected errors and no test discovered after project is moved on disk
**Describe the bug**
When a project with an existing .pytest_cache folder is moved to a different location on disk and opened in VS, tests are not discovered and a `KeyNotFoundException` error appears in the Tests output window.

**Steps to Reproduce**
1. Open a project and enable pytest. Confirm tests are discovered.
2. Close the project
3. Move the solution/project folder to a different location on disk
4. Load the solution/project from the new location

**Expected behavior**
Tests are discovered successfully

**Additional context and screenshots**
```
[7/11/2019 2:25:07.188 PM Error] System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Immutable.ImmutableDictionary`2.get_Item(TKey key)
   at Microsoft.VisualStudio.TestWindow.Host.VsTestRunSession.<>c__DisplayClass17_0.<AddTestCases>b__0(TestCase test)
   at System.Linq.Enumerable.WhereSelectListIterator`2.MoveNext()
   at Microsoft.VisualStudio.TestWindow.Host.TestRunSession.AddTestCases(IEnumerable`1 testCases)
   at Microsoft.VisualStudio.TestWindow.Host.TestPlatformProvider.<>c__DisplayClass41_0.<HandleDiscoveredTests>b__0()
   at Microsoft.VisualStudio.TestWindow.Extensibility.ILoggerExtensions.CallInternalWithLogging(ILogger2 log, Action action, Boolean shouldThrow)
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at Microsoft.VisualStudio.Telemetry.WindowsErrorReporting.WatsonReport.GetClrWatsonExceptionInfo(Exception exceptionObject)
```

Not sure if we can detect that the cache is out of date, or maybe tell pytest not to use the cache? That may have too big of a perf hit?

The symptom and error message don't make it very obvious as to what the problem is.

### PR DESCRIPTION
**Pull request by bschnurr**
Fix #5425

### CODE REVIEW

File: Python/Product/TestAdapter.Executor/Pytest/TestDiscovererPytest.cs [hunk 1]
```diff
@@ -105,8 +105,9 @@ ITestCaseDiscoverySink discoverySink
                 foreach (PytestTest test in result.Tests) {
                     try {
                         (string parsedSource, int line) = test.ParseSourceAndLine();
-                        var parsedFullSourcePath = Path.IsPathRooted(parsedSource) ? parsedSource : Path.Combine(_settings.ProjectHome, parsedSource);
-                     
+                        // Note: we use _settings.ProjectHome and not result.root since it is being lowercased
+                        var parsedFullSourcePath = Path.IsPathRooted(parsedSource) ? parsedSource : PathUtils.GetAbsoluteFilePath(_settings.ProjectHome, parsedSource);
+
```
Comment (bschnurr) [inline Python/Product/TestAdapter.Executor/Pytest/TestDiscovererPytest.cs]: also changed Path.Combine to PathUtils.GetAbsoluteFilePath, but its returning the same result

**Comment by bschnurr** [comment]
image

File: Python/Product/Common/Strings.resx [hunk 2]
```diff
@@ -2957,8 +2957,10 @@ Please specify at least one package.</value>
     <value>Discovery results file '{0}' was not found.</value>
   </data>
   <data name="ErrorTestContainerNotFound" xml:space="preserve">
-    <value>Test container was not found for {0}.</value>
-    <comment>{0} is the test info</comment>
+    <value>Test container was not found under folder {0} 
+for test {1}.</value>
+    <comment>{0} is the root directory
```
Comment (bschnurr) [inline Python/Product/Common/Strings.resx]: maybe its better to not use a line break here?

Comment (huguesv) [reply Python/Product/Common/Strings.resx]: I'd vote for no line break. User can Toggle Word Wrap on the output window.

Comment (bschnurr) [reply Python/Product/Common/Strings.resx]: ok. updating

**Pull request merged**